openwisp_radius/base/models.py

```diff
@@ -1295,6 +1307,22 @@ def _clean_password_reset_url(self):
         ):
             self.password_reset_url = None
 
+    def _clean_sms_message(self):
+        if self.sms_message and ('{code}' not in self.sms_message):
+            raise ValidationError(
+                {
+                    'sms_message': _(
+                        'The SMS message must contain the "{{code}}" '
+                        'placeholder, eg: {}.'.format(app_settings.SMS_MESSAGE_TEMPLATE)
+                    )
+                }
+            )
+        if (
+            not self.sms_message
+            or self.sms_message == app_settings.SMS_MESSAGE_TEMPLATE
+        ):
+            self.sms_message = None
```

Comment:
isn't this slightly redundant with https://github.com/openwisp/openwisp-radius/pull/436?